fastlane/resources/values/strings.xml

```diff
@@ -2429,6 +2444,18 @@
     <string name="domain_registration_state_picker_dialog_title">Select State</string>
     <string name="domain_registration_registering_domain_name_progress_dialog_message">Registering domain name…</string>
 
+    <string name="domains_primary_domain">PRIMARY SITE ADDRESS</string>
+    <string name="domains_primary_domain_address">%s</string>
+<!--    <string name="domains_primary_domain_blurb">Your primary site address is what visitors will see in browser address bar when they visit your website.</string>-->
+<!--    <string name="domains_redirected_domains">REDIRECTED DOMAINS</string>-->
+<!--    <string name="domains_redirected_domains_blurb">Redirected domains are domains that you own and redirect to your site at %s. Anyone visiting your redirected domains will land on your site. Learn more.</string>-->
+<!--    <string name="domains_manage_domains">Manage Domains</string>-->
+<!--    <string name="domains_paid_plan_claim_your_domain_title">Claim your free domain</string>-->
+<!--    <string name="domains_paid_plan_claim_your_domain_caption">You have a free one-year domain registration included with your domain</string>-->
+    <string name="domains_free_plan_get_your_domain_title">Get your domain</string>
+<!--    <string name="domains_free_plan_get_your_domain_caption">Redirect a custom domain to your site at %s</string>-->
+<!--    <string name="domains_free_plan_redirect_learn_more">Learn more about domain redirects</string>-->
```

Comment:
Was surprised by these commented elements, but I can see they appear like that in the source, too: 

https://github.com/wordpress-mobile/WordPress-Android/blob/917938ebc9738cf8c22173fb6624ac20738b7898/WordPress/src/main/res/values/strings.xml#L2449-L2457

I assume the GlotPress upload tool knows how to distinguish a comment like this from one for the translators, I'm guessing because of the `translators:` prefix at the start of the comment content, and this won't be a problem. 👍

Reply:
Good catch. That's a change made by @ravishanker in https://github.com/wordpress-mobile/WordPress-Android/commit/fe55b5eae960733d27cc03fa7f27cd40982ab631, maybe they have an insight about why those were commented out (and not e.g. completely removed)?

But yeah I think GlotPress is smart enough to exclude those comments indeed.